src/main/js/PipelineEditorLink.jsx

```diff
@@ -34,7 +36,7 @@ class PipelineEditorLink extends React.Component {
 
         return (
             <Link className="pipeline-editor-link" to={baseUrl} title="Edit">
-                <Icon icon="ImageEdit" size={24} color="rgba(53, 64, 82, 0.25)" hoverColor="#4a90e2" />
+                <Icon icon="ImageEdit" size={24} color={editIconColor} hoverColor={editIconHoverColor} />
```

Comment:
Ok, added this ticket to address changing all icon styles to css: https://issues.jenkins-ci.org/browse/JENKINS-46348